packages/inferno/src/DOM/rendering.ts

```diff
@@ -81,6 +81,7 @@ export function render(
   if (rootInput && rootInput.flags & VNodeFlags.Component) {
     return rootInput.children;
   }
+  return;
```

Comment:
What is the point in this? return as last statement? That would only increase bundle size

Reply:
Point it to not have an inplicit 'return undefined' statement, so a correct explicit one should be used. I'm not sure which is the correct one...